apps/docs/src/app/platform/component-docs/platform-approval-flow/platform-approval-flow-examples/platform-approval-flow-example.component.html

```diff
@@ -10,6 +13,8 @@
     (nodeClick)="nodeClick($event)"
     (afterNodeAdd)="afterNodeAdd($event)"
     (afterNodeEdit)="afterNodeEdit($event)"
+    (approvalFlowNodesUpdated)="approvalFlowNodesUpdated($event)"
```

Comment:
rename to `nodeUpdate` or `valueChange` so we can keep it similar to the other components

Reply:
done